entities/weapons/pocket/cl_menu.lua

```diff
@@ -32,14 +32,26 @@ function DarkRP.openPocketMenu()
     if not wep:IsValid() or wep:GetClass() ~= "pocket" then return end
     if not pocket then pocket = {} return end
     if table.Count(pocket) == 0 then return end
-    frame = vgui.Create("DFrame")
-
+    frame = vgui.Create( "DFrame" )
```

Comment:
The DarkRP codebase consistently doesn't use airy coding (no space between parentheses and args)